### ISSUE TITLE
Accessibility concerns

### ISSUE THREAD

**Issue by jcklpe** [LemmyNet/lemmy-ui#804]
Using the [Wave accessibility checker tool](https://wave.webaim.org/) we get a number of issues highlighted:
![image](https://user-images.githubusercontent.com/9451726/192414408-ab414a2a-00f1-46ff-8e49-d9d0be40ce11.png)

It might be worth finding a contributor who specializes in accessibility to help with this stuff. A lot of it is contrast errors and lacking ARIA labeling.

**Comment by dessalines** [comment]
Thanks for finding these, they shouldn't take too long to fix.

**Comment by dessalines** [comment]
Stale issue
